Kitodo/src/main/webapp/WEB-INF/web.xml

```diff
@@ -88,15 +88,6 @@
         <param-value>blueSky</param-value>
     </context-param>
 
-    <filter>
-        <filter-name>Proxy</filter-name>
-        <filter-class>net.sf.j2ep.ProxyFilter</filter-class>
-        <init-param>
-            <param-name>dataUrl</param-name>
-            <param-value>/WEB-INF/reverse-proxy.xml</param-value>
-        </init-param>
-    </filter>
-
     <filter-mapping>
```

Comment:
Is this entry still needed after you removed the proxy?

Reply:
You're right. Thanks for the hint